main.js

```diff
@@ -27,6 +28,11 @@ class Game {
         }
     }
     imageOnClick(event) {
+        if(!this.timer || !this.timer.isStarted) {
+            this.timer=new Timer();

```

Comment:
Let's move instance creation to the constructor and then simplify the code
```js
if(!this.timer.isStarted) {
  this.timer.startTimer();
}
```

Reply:
@probil, ok.